service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/spring/KeycloakServiceImpl.java

```diff
@@ -228,6 +211,18 @@ private UserRepresentation create(String userName, String password, boolean enab
 
 			user.setCredentials(Arrays.asList(getUserCredentials(password, false)));
 		}
+		if (userTenantInfos == null) {
+			logger.error("create(username={}) no userTenantInfos received", userName);
+			throw new IllegalStateException("Missing userTenantInfos for username " + userName);
+		} else {
+			try {
+				user.singleAttribute(USER_ID_PROPERTY, toTenantInfosString(userTenantInfos));
+			} catch (JsonProcessingException e) {
+				logger.error("create(username={}) Failed ({}) to write identifier={} : {}, username",
```

Comment:
log message missing an argument? 
it seems that `username` appearing in the end of the string should have been an argument but was instead swallowed into the text block